tests/suites/test_suite_psa_crypto.function

```diff
@@ -2426,7 +2426,8 @@ void cipher_encrypt( int alg_arg, int key_type_arg,
     total_output_length += function_output_length;
 
     status = psa_cipher_finish( &operation,
-                                output + total_output_length,
+                                ( output_buffer_size == 0 ? NULL :
```

Comment:
Why not just testing "output" ?
( output == NULL ? NULL : ... )
With the test on output_buffer_size we have to dig into the code above to understand what is going on.

Reply:
Because if the buffer size is nonzero, `output` has no business being null. And conversely, if the buffer size is zero, this makes it clearer that we're testing the function with NULL (it is the case anyway since in all the places I changed, `output` comes from `ASSERT_ALLOC` which guarantees NULL for a 0-sized buffer, but this wouldn't be the case for a buffer coming from malloc/calloc).

Reply:
ok yes I agree no strong cut here. Fine by me and thanks for your reply.